src/components/molecules/ImportModal.unit.ts

```diff
@@ -185,10 +152,105 @@ describe("@components/molecules/RoomModal", () => {
 		wrapper.setData({ step: 2 });
 		await wrapper.vm.$nextTick();
 
-		const btnConfirm = wrapper.find(".dialog-confirmed");
+		const btnConfirm = wrapper.find(".dialog-next");
 		expect(btnConfirm.element.innerHTML).toContain("Weiter");
```

Comment:
instead of hard-coding text, I would use translate function. This means if text is updates in lang file, the test will pass
```suggestion
               const nextText = wrapper.vm.$i18n.t("pages.rooms.importCourse.btn.continue");
		expect(btnConfirm.element.innerHTML).toContain(nextText);
```